planetarium/NineChronicles
# Add stage ranking.

### Description

1. Add stage ranking on bottom menu rank window.

### How to test

1. Enter Ranking by clicking `Ranking` button on bottom menu.
2. Check if stage ranking is working properly.

### Required Reviewers

@planetarium/9c-dev @Namyujeong @planetarium/9c-backend (optional)
